creation/lib/cvWParamDict.py

```diff
@@ -851,11 +851,15 @@ def populate_frontend_descript(work_dir, frontend_dict, active_sub_list, params)
         frontend_dict.add("MonitoringWebURL", params.web_url.replace("stage", "monitor"))
 
     # TODO: refcred (refactoring of credentials) remove proxy requirement, replace w/ any credential, maybe ID
-    if params.security.classad_proxy is None:
-        raise RuntimeError("Missing security.classad_proxy")
-    params.subparams.data["security"]["classad_proxy"] = os.path.abspath(params.security.classad_proxy)
-    if not os.path.isfile(params.security.classad_proxy):
-        raise RuntimeError("security.classad_proxy(%s) is not a file" % params.security.classad_proxy)
+    # for now, just allow absence of proxies in face of JWT hedgemony
+    # if params.security.classad_proxy is None:
+    #    raise RuntimeError("Missing security.classad_proxy")
+    try:
+        params.subparams.data["security"]["classad_proxy"] = os.path.abspath(params.security.classad_proxy)
+    except:
+        params.subparams.data["security"]["classad_proxy"] = None
+    # if not os.path.isfile(params.security.classad_proxy):
+    #    raise RuntimeError("security.classad_proxy(%s) is not a file" % params.security.classad_proxy)
```

Comment:
Try to avoid generic `except`, if there is a name and the file is not there, it is still an error. Something like:
```
    if params.security.classad_proxy is None:
        params.subparams.data["security"]["classad_proxy"] = None
    else:
        params.subparams.data["security"]["classad_proxy"] = os.path.abspath(params.security.classad_proxy)
        if not os.path.isfile(params.security.classad_proxy):
            raise RuntimeError("security.classad_proxy(%s) is not a file" % params.security.classad_proxy)
```

Reply:
done